travelhook/database.py

```diff
@@ -654,6 +658,129 @@ def get_db_composition(self):
             ).fetchone()["newpatch"]
             self.write_patch(json.loads(newpatch))
 
+    def get_vagonweb_composition(self):
+        if "composition" in self.status:
```

Comment:
ähnlich wie beim (erst heute entsprechend gefixtem) `get_db_composition` ein "failedcomposition-vagonweb" abfragen bzw bei fehlerhaftem request in den patch setzen, damit vagonweb nur einmal angefragt wird
https://github.com/deingithub/travelhook/blob/701acef68f57629355da8f32d541376402415879/travelhook/database.py#L538-L550